samples/SampleTypeConversionTest.py

```diff
@@ -0,0 +1,369 @@
+#
+# Copyright 2019 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Starting from Python 3.8 DLL search policy has changed.
+# We need to add path to CUDA DLLs explicitly.
+import sys
+
+import PyNvCodec as nvc
+import os
+import tqdm
+import argparse
+from pathlib import Path
+from enum import Enum
+
+import numpy as np
+import logging
+
+logger = logging.getLogger(__file__)
+
+if os.name == "nt":
+    # Add CUDA_PATH env variable
+    cuda_path = os.environ["CUDA_PATH"]
```

Comment:
Can we remove that? On Windows we're bundling all `npp` and `cudart` libs which made this section necessary. Does it work without it?

Reply:
will try this in new feature branch, marking this as resolved for now